src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -9,68 +9,201 @@ import { localize } from 'vs/nls';
-						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
-						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
-						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
-						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
-					</ul>
-				</div>
-				<div class="section deploy">
-					<h2 class="caption">${escape(localize('welcomePage.deploy', "Deploy"))}</h2>
-					<!--The resourceDeployment.new command can be removed if this option is removed from the welcome page-->
-					<ul>
-						<li><a href="command:resourceDeployment.new">${escape(localize('welcomePage.newDeployment', "New Deployment…"))}</a></li>
-					</ul>
+		<div class="ads_homepage splash">
+			<div class="gradient">
+				<div class="preview_text tool_tip">
+					<div class="tool_tip__container" id="tool_tip__container--desktop">
+						<a aria-describedby="tooltip__text--desktop" id="preview_link--desktop" class="preview_link" tabindex="0" name="preview"><p>Preview</p><i class="icon--info themed_icon"></i></a>
+						<span role="tooltip" id="tooltip__text--desktop" class="tool_tip__text" aria-hidden="true">
+							<h3 tabindex="0" class="preview_tooltip__header">This page is in preview</h3>
+							<p tabindex="0" class="preview_tooltip__body">Preview features introduce new functionalities that are on track to becoming a permanent part the product. They are stable, but need additional accessibility improvements. We welcome your early feedback while they are under development.</p>
+						</span>
+					</div>
+					<div class="tool_tip__container" id="tool_tip__container--mobile">
+						<a aria-haspopup="true" class="preview_link" tabindex="0" id="preview_link--mobile" name="previewMobile"><p>Preview</p><i class="icon--info themed_icon"></i></a>
+					</div>
 				</div>
-				<div class="section recent">
-					<h2 class="caption">${escape(localize('welcomePage.recent', "Recent"))}</h2>
-					<ul class="list">
-						<!-- Filled programmatically -->
-						<li class="moreRecent"><a href="command:workbench.action.openRecent">${escape(localize('welcomePage.moreRecent', "More..."))}</a><span class="path detail if_shortcut" data-command="workbench.action.openRecent">(<span class="shortcut" data-command="workbench.action.openRecent"></span>)</span></li>
-					</ul>
-					<p class="none detail">${escape(localize('welcomePage.noRecentFolders', "No recent folders"))}</p>
+				<div id="preview_modal" class="modal" aria-modal="true" aria-hidden="true">
+					<div class="modal_content">
+						<span class="close_icon">x</span>
+						<h3 tabindex="0" class="preview_modal__header">This page is in preview</h3>
```

Comment:
All text on this page should be localized

Reply:
resolved